thali/thalicryptomanager.js

```diff
@@ -0,0 +1,102 @@
+'use strict';
+
+var crypto = require('crypto');
+var fs = require('fs');
+var path = require('path');
+
+var pkcs12FileName = '/pkcs12.pfx';
+var password = 'password';
```

Comment:
I'll go file an issue that we really should be using the key stores on both iOS and Android. But keep in mind that this only provides a nominal security benefit since no secure hardware is being used. Instead what it does it store the keys in an encrypted file based off the device's key. This does provide some (tiny) level of protection for unencrypted devices. For devices that encrypt their drives it provides no meaningful protection. As such I think adding in support for local keystores is low priority.